config/data_sources/consultation/global.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
This file is generated by a developer and can be modeled after other existing data sources of the same type such as CSV or external database.